GHzDACs/GHzAD_v7.md

```diff
@@ -0,0 +1,437 @@
+# Ethernet Commands for GHzADC
+
+John Martinis
+
+## General specifications
+
+FPGA interface for GHzADC.
+Prototype is version/build 1, 2 and 6.
+Revised are others.
+DO NOT MIX as FPGA pinouts changed.  
+
+The two AD converters on the board are each 8 bits at 1 Gs/s.
+To simplify computation, data from adjacent times are summed, giving a net data rate at 500 Ms/s, 9 bits.
+With two (I and Q) channels, this gives a system bandwidth of +- 250 MHz.  
```

Comment:
Add a note that this is not particularly effective at antialiasing, and you should avoid having signal in the 250-500 MHz band.